### PR TITLE
M365 May 2022 Update

### PR DESCRIPTION
**Pull request by mikeebowen**
- Added PowerPointCommentPart relationship to SlidePart (https://github.com/OfficeDev/Open-XML-SDK/pull/1137)
  - This deprecates the relationship between PowerPointCommentPart and PresentationPart, which will be removed in a future version
- Removed dogfood Reactions feature

### CODE REVIEW

File: test/DocumentFormat.OpenXml.Packaging.Tests/ParticleTests.cs [hunk 1]
```diff
@@ -223,6 +223,7 @@ private void AssertEqual(Dictionary<Type, VersionCollection<ParticleConstraint>>
             var tmp = Path.GetTempFileName();
 
             _output.WriteLine($"Writing output to {tmp}");
+            _output.WriteLine("Expected output in DocumentFormat.OpenXml.Packaging.Tests.data.Particles.json");
```
Comment (twsouthwick) [inline test/DocumentFormat.OpenXml.Packaging.Tests/ParticleTests.cs:226]:
```suggestion
            _output.WriteLine(@"Expected output in data\Particles.json");
```

that way you can actually find the file

File: src/DocumentFormat.OpenXml/OpenXmlCompositeElement.cs [hunk 2]
```diff
@@ -9,6 +9,7 @@
 using System.Diagnostics.CodeAnalysis;
 using System.Globalization;
 using System.IO;
+using System.Reflection;
```
Comment (twsouthwick) [inline src/DocumentFormat.OpenXml/OpenXmlCompositeElement.cs:12]:
```suggestion
```

unnecessary

**Review by twsouthwick — APPROVED** [review]
LGTM

**Comment by mikeebowen** [comment]
> How did you add the shim? I don't see custom code for that anywhere

It's on a branch [on my fork](https://github.com/mikeebowen/Open-XML-SDK/blob/add-extension-list-utils/src/DocumentFormat.OpenXml/Schema/Office2021/Word/CommentsExt/CommentsExt.cs)

It's not included in the PR, because the Schema for Reaction is in dogfood and will change, so we don't want to release something with the current Reaction, because it would allow users to create invalid packages. Without Reaction, there are no children of the CommentsExtensibleExtensionList, so we don't need to shim ExtensionList, yet. When the schema for Reaction is finalized, we'll have the shim ready.

**Review by twsouthwick — APPROVED** [review]

**Pull request merged**